### PR TITLE
Test unicode handling in context of SerDes 

### PR DESCRIPTION
**Pull request by 650elx**
Related fixtures:
https://github.com/sinch/sinch-sdk-internal-specs/blob/numbers_fixtures/fixtures/python/verification-api/mappings/verification_v1_verifications-f0c31c63-19fc-4062-8bbd-d0e96b3468e8.json

https://github.com/sinch/sinch-sdk-internal-specs/blob/numbers_fixtures/fixtures/python/verification-api/mappings/verification_v1_verifications_id_01919a50-b75b-cedb-5080-216b829e92e9-0a03359a-6c0e-4b65-aa92-f40a74e6d19a.json

### CODE REVIEW

File: tests/integration/test_transport_unicode_handling.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    verification_response = sinch_client_sync.verification.verifications.start_flash_call(
+        identity={
+            "type": "number",
+            "endpoint": phone_number
+        },
+        reference=UNICODE_POTATO,
+        dial_timeout=5
+    )
+
+    assert isinstance(verification_response, StartFlashCallInitiateVerificationResponse)
```
Comment (Dovchik) [inline tests/integration/test_transport_unicode_handling.py:22]: nit
I suppose you can check send and receive in just one function and one mock

**Review by Dovchik — COMMENTED** [review]
What's the motivation for removing `run-ws` workflow, can you update the PR description about that?

File: tests/integration/test_transport_unicode_handling.py (hunk 1 above)
Comment (Dovchik) [reply tests/integration/test_transport_unicode_handling.py:22]: Ah, also, would be nice to see a ref to the mock in PR description

**Comment by 650elx** [comment]
> What's the motivation for removing `run-ws` workflow, can you update the PR description about that?

I've forgot to update target branch with the latest changes present in main. Fixed

File: tests/integration/test_transport_unicode_handling.py (hunk 1 above)
Comment (650elx) [reply tests/integration/test_transport_unicode_handling.py:22]: > nit I suppose you can check send and receive in just one function and one mock

Wiremock is doing that behind the scenes :)

Comment (650elx) [reply tests/integration/test_transport_unicode_handling.py:22]: > Ah, also, would be nice to see a ref to the mock in PR description

👍

Comment (Dovchik) [reply tests/integration/test_transport_unicode_handling.py:22]: > Wiremock is doing that behind the scenes :)

Always forgot that 😅

**Review by Dovchik — APPROVED** [review]

**Pull request merged**